CHANGELOG.md

```diff
@@ -0,0 +1,29 @@
+# libDaisy Changelog
+
+## Unreleased
+
+* Update TODO comment in uart.h to reflect most recent uart update.
+* Add MidiEvent and related data to daisy namespace.
+* Update qspi API from C to C++.
```

Comment:
Both of the above lines should have a "BREAKING CHANGE: " prefix, IMHO

Reply:
Good call! Yeah, I figured it'd be obvious by the migration notes below, but as clear as we can be about that, the better.